django_project/realtime/tasks/realtime/test/test_realtime_tasks.py

```diff
@@ -33,10 +33,12 @@
     'Realtime Worker needs to be run')
 class TestAshTasks(test.LiveServerTestCase):
 
-    def setUp(self):
+    @classmethod
+    def setUpClass(cls):
+        super(TestAshTasks, cls).setUpClass()
         app_config = apps.get_app_config('realtime')
-        app_config.load_volcano_fixtures()
-        app_config.load_test_users()
+        app_config.create_rest_group()
```

Comment:
I hate django tests...
I had to recreate every user group because it will be deleted in the next tests.
I tried using fixtures but the id can't be controlled easily, so I give up.
Also, app_config.ready() doesn't get executed automatically in test environment, which is frustrating...